lib/ship.rb

```diff
@@ -0,0 +1,23 @@
+class Ship
+
+  attr_reader :name, :length, :health
+
+  def initialize(name_parameter, length_parameter)
+    @name = name_parameter
+    @length = length_parameter
+    @health = @length
```

Comment:
I don't know for certain atm, but I don't think @health needs to be set to @length - Tried with @health = length and it seems to work fine